specs/protocol/superchain-upgrades.md

```diff
@@ -256,8 +256,7 @@ but the matching L1-origin information may not be present at the time of activat
   [Governance Proposal](https://gov.optimism.io/t/final-upgrade-proposal-2-canyon-network-upgrade/7088).
 - `v5.0.0`: 2024 Feb 22nd - Delta network upgrade (Span Batches).
   [Governance Proposal](https://gov.optimism.io/t/final-upgrade-proposal-3-delta-network-upgrade/7310).
-- `v6.0.0`: 2024 Mar 14th - _Proposed_ Ecotone network upgrade (4844 Blob Batches + Cancun).
-  Still under review by governance.
+- `v6.0.0`: 2024 Mar 14th - Ecotone network upgrade (4844 Blob Batches + Cancun).
```

Comment:
This would be good as its own PR

Reply:
[1c4cea8](https://github.com/ethereum-optimism/specs/pull/115/commits/1c4cea877aba750787bdac5419d32041e53a74de)